spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../person'
+
+describe Person do
+  before(:each) do
+    @person = Person.new(20, 'Joffrey')
+  end
+
```

Comment:
- [ ] According to the requirements, you need to test all the methods in the class. This means you need to add tests for the following methods 👇 
1. `add_rental`
2. `can_use_services`